src/io/flutter/sdk/FlutterSettingsConfigurable.form

```diff
@@ -88,6 +88,15 @@
               <toolTipText value="Enables verbose logging (this can be useful for diagnostic purposes)."/>
             </properties>
           </component>
+          <component id="984f6" class="javax.swing.JCheckBox" binding="myUseBazelByDefaultCheckBox">
+            <constraints>
+              <grid row="3" column="0" row-span="1" col-span="1" vsize-policy="0" hsize-policy="3" anchor="8" fill="0" indent="0" use-parent-layout="false"/>
+            </constraints>
+            <properties>
+              <text value="Use Bazel instead of Pub by default"/>
```

Comment:
Do we need a UI setting for this? Should this be controlled on a per app basis, or just via the registry setting?

Reply:
I think ideally, we will be able to determine this and other sensible defaults for Bazel at startup time and apply them in bulk.

Until adding that, this is useful.  I filed #3437 to do this, and I'll submit this change for now.